label_studio/khan/settings/base.py

```diff
@@ -0,0 +1,49 @@
+"""This is the settings file that serves as the base for all other khan run environments.
+It inherits from the label-studio core base settings, and all khan's settings files should inherit from this.
+"""
+from core.settings.base import *
+
+# Make sure our custom django app is installed
+INSTALLED_APPS.extend([
```

Comment:
_django n00b wondering_: No action required.  I wonder if this is how we get the characteristic of that `LS_TOKEN > iap-jwt` authentication, since both the `rest_framework` and the`label_manager` (?) app would run first?



[1] https://github.com/Khan/label-studio/blob/develop/label_studio/core/settings/base.py#L178

Reply:
IIRC, the order of installed apps doesn't really matter, what does matter is the order of `MIDDLEWARE` and `REST_FRAMEWORK['DEFAULT_PERMISSION_CLASSES']` and `REST_FRAMEWORK['DEFAULT_AUTHENTICATION_CLASSES']` as well as, in the internals of Django/DRF the order those two things call each other/deferentially treat the other. ie: does the drf internals respect the user added from JWT? or does it prefer it's own authentication if a token is provided. And, with at least anecdotal evidence, it seems that drf prefers the user from the auth token, when provided, but will fall back to the django user from the middleware.